docs/addons/keda.md

```diff
@@ -0,0 +1,146 @@
+}
+const addOn = new blueprints.addons.KedaAddOn(kedaParams)
+
+const blueprint = blueprints.EksBlueprint.builder()
+  .addOns(addOn)
+  .build(app, 'my-stack-name');
+```
+
+## Configuration Options
+
+- `version`: Version fo the Helm Chart to be used to install Kubevious
+- `kedaOperatorName`: Name of the KEDA operator
+- `enableIRSA`: Specifies whether a service account should be created by by CDK with IRSA. If provided false, Service Account will be created by Keda without IRSA
+- `createServiceAccount`: Specifies whether a service account should be created by Keda. 
+- `kedaServiceAccountName`: The name of the service account to use. If not set and create is true, a name is generated.
+- `podSecurityContextFsGroup`: PodSecurityContext holds pod-level security attributes and common container settings. fsGroup is a special supplemental group that applies to all containers in a pod. This is an optional attribute, exposed directly inorder to make KedaScalar workaroud - https://github.com/kedacore/keda/issues/837#issuecomment-789037326
+- `securityContextRunAsUser`: SecurityContext holds security configuration that will be applied to a container. runAsUser is the UID to run the entrypoint of the container process. This is an optional attribute, exposed directly inorder to make KedaScalar workaroud - https://github.com/kedacore/keda/issues/837#issuecomment-789037326
+- `securityContextRunAsGroup`: SecurityContext holds security configuration that will be applied to a container. rusAsGroup The GID to run the entrypoint of the container process. This is an optional attribute, exposed directly inorder to make KedaScalar workaroud - https://github.com/kedacore/keda/issues/837#issuecomment-789037326
+
+- `values`: Arbitrary values to pass to the chart. Refer to the Kubevious [Helm Chart documentation](https://github.com/kubevious/helm) for additional details
+
+## Validation
+
+To validate that Kubevious is installed properly in the cluster, check that the Kubevious deployments,
+services and stateful sets are running.
+
+```shell
+  kubectl get deployments -n keda  
+```
+There should be 2 deployments are created once for operator/agent and another for metrics server
+
+```shell
+  kubectl get pods -n keda  
+```
+There should be 2 pods one startin with name keda-operator- (Eg: "keda-operator-56bb8ddd5b-7fqb7")
+Verify IRSA is working properly
+```shell
+kubectl describe pods keda-operator-8c5967bcd-vlhpd  -n keda | grep -i aws  
+```
+
+```shell
+  kubectl get sa -n keda
+  kubectl describe sa keda-operator -n keda  
+```
+There should be an Service Account with name KedaOperator created and you can see an annotation on Service Account which have the AWS IAM Role ARN ( with required access for AWS Services like AWS SQS)
+
+## Testing
+
+We will test AWS SQS Scalar here
+1) Create SQS Queue in desired region
+```shell
+    aws sqs create-queue --queue-name keda-test --region REGION_NAME --output json
```

Comment:
Should REGION_NAME be replaced with AWS_REGION for consistency with the rest?

Reply:
Fixed